packages/wonder-blocks-labeled-field/src/components/error-helper-text.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import WarningCircle from "@phosphor-icons/core/bold/warning-circle-bold.svg";
+import {PhosphorIcon} from "@khanacademy/wonder-blocks-icon";
+import {HelperText} from "./helper-text";
+import theme from "../theme";
+
+type Props = {
+    message?: React.ReactNode;
+    id?: string;
+    testId?: string;
+    show: boolean;
+    iconAriaLabel?: string;
+};
+
+export const ErrorHelperText = (props: Props) => {
+    const {message, id, testId, show, iconAriaLabel = "Error: "} = props;
+
+    return (
+        <HelperText
+            id={id}
+            testId={testId}
+            styles={{root: {color: theme.error.color.foreground}}}
+            message={show && message}
+            icon={
+                show && (
+                    <PhosphorIcon
+                        icon={WarningCircle}
+                        aria-label={iconAriaLabel}
+                    />
+                )
+            }
+            // We use aria-live="assertive" for the error so that it is
+            // immediately announced and the user can address the issue
+            // before submitting the form. We use aria-live=assertive
+            // instead of role=alert because Safari + VoiceOver would
+            // not read out the error when focused on if the element
+            // referenced by the aria-describedby had role="alert".
+            aria-live="assertive"
+            // We add aria-atomic=true so that any updates to the error
+            // is announced
+            aria-atomic="true"
+        />
+    );
+};
```

Comment:
Overall, I also like breaking out the helper text into smaller components to simplify the LabeledField component. We can also export it and they can be re-used in other places too if there are custom implementations of things that would have error text.

We can break out the helper text into smaller components regardless of which approach we take!